tasks/main.yml

```diff
@@ -60,12 +60,22 @@
 
   register: _zookeeper_conf
 
+- name: Check service status
```

Comment:
If a service is down for an identified reason (disk broken or corrupted) this change will start it.
I think, it's too risky

With this change, you can't write
```
 ##### IMPACT
N/A
```